table_foosball/database.rb

```diff
@@ -0,0 +1,117 @@
+require_relative "./text_tools"
+
+class Database
+  include TextTools
+
+  def initialize(name)
+    @name = name
+    @filename = "./#{name}.sdb"
+  end
+
+  def clear
+    File.truncate("./#{@name}", 0)
+  end
+
+  def save(objs)
+    file = File.open(@filename, "r+")    
+    file.rewind
```

Comment:
`"r+"` doesn't truncate the existing file to length 0 like `"w"` does, which is actually a discrepancy. The file should be completely overwritten for each call to `#save`, I'd say. Thanks for pointing it out!